app-ios/project.yml

```diff
@@ -0,0 +1,425 @@
+          "tutanota-web",
+          "Sources/Crypto/liboqs",
+          "Sources/Crypto/phc-winner-argon2",
+          "Sources/sdktest.swift"
+        ]
+        compilerFlags: [
+          "-I$(BUILD_DIR)/liboqs-include",
+          "-Itutanota/Sources/Crypto/phc-winner-argon2/include"
+        ]
+        # XCode will not resolve symbolic links when adding a folder reference, so we will just use a relative path
+      - path: "../build"
+        name: "build"
+        type: "folder"
+        buildPhase: "resources"
+    settings:
+      base:
+        PRODUCT_MODULE_NAME: "tutanota"
+        ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES: "YES"
+        LD_RUNPATH_SEARCH_PATHS: "@executable_path/Frameworks"
+        INFOPLIST_FILE: "tutanota/Info.plist"
+        SWIFT_OBJC_BRIDGING_HEADER: "tutanota/Sources/tutanota-Bridging-Header.h"
+        HEADER_SEARCH_PATHS: [
+          "$(BUILD_DIR)/liboqs-include",
+          "${PROJECT_DIR}/tutanota/include",
+          "tutanota/Sources/Crypto/phc-winner-argon2/include"
+        ]
+        CLANG_CXX_LANGUAGE_STANDARD: "gnu++14"
+        GCC_PREPROCESSOR_DEFINITIONS: [
+          "SQLITE_HAS_CODEC=1",
+          "OQS_ENABLE_KEM_kyber_1024=1",
+          "$(inherited)"
+        ]
+        OTHER_CFLAGS[arch=*]: [
+          "-DSQLITE_HAS_CODEC",
+          "-DSQLITE_TEMP_STORE=3",
+          "-DSQLCIPHER_CRYPTO_CC",
+          "-DNDEBUG",
+          "-DKYBER_K=4",
+          "-DOQS_VERSION_TEXT=\\\"whatever\\\"",
+          "-DOQS_ENABLE_KEM_kyber_1024=1",
+          "-DOQS_DIST_BUILD=1",
+          "-DOQS_ENABLE_KEM_KYBER=1"
+        ]
+        SUPPORTS_XR_DESIGNED_FOR_IPHONE_IPAD: "NO"
+        SUPPORTS_MAC_DESIGNED_FOR_IPHONE_IPAD: "NO"
+      configs:
+        Debug Development:
+          PRODUCT_NAME: "Tuta D"
+        Debug Staging:
+          PRODUCT_NAME: "Tuta Test"
+        Debug Production:
+          PRODUCT_NAME: "Tuta Mail"
+        Release Development:
+          PRODUCT_NAME: "Tuta D"
+        Release Staging:
+          PRODUCT_NAME: "Tuta Test"
+          PROVISIONING_PROFILE_SPECIFIER: "match AdHoc de.tutao.tutanota.test"
+          PROVISIONING_PROFILE_SPECIFIER[sdk=iphoneos*]: "match AdHoc de.tutao.tutanota.test 1716556941"
+        Release Production:
+          PRODUCT_NAME: "Tuta Mail"
+      groups:
+        - "codeSigning"
+    dependencies:
+      - package: Atomics
+      - sdk: CryptoTokenKit.framework
+        embed: false
+        # FIXME make `CryptoTokenKit.framework` optional
+      - package: DictionaryCoding
+      - sdk: Foundation.framework
+        embed: false
+      - framework: libcrypto.xcframework
+        embed: false
+      - sdk: Security.framework
+        embed: false
+      - target: TutanotaNotificationExtension
+        codeSign: false
+      - target: TutanotaSharedFramework
+      - target: TutanotaShareExtension
+        codeSign: false
+      - framework: tutasdk.framework
+        embed: false
+      - target: oqs
+      - target: argon2
+  oqs:
```

Comment:
We could move oqs and argon into their own projects linked with the workspace. That would be cleaner as they are dependencies rather than a true part of the client.